docs/beamrunner/beamrunner-5b-objstor.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title:  Using the IBM Object Storage OpenStack Swift for Bluemix service with IBM Streams Runner for Apache Beam
+navtitle: Using Bluemix Object Storage
+description:  You can use the IBM® Streams Runner for Apache Beam `FileStreamSample` sample application to learn how to use Bluemix object storage for file input and output.
+weight:  10
+published: true
+tag: beam
+prev:
+  file: beamrunner-5a-io
+  title: I/O options
+next:
+  file: beamrunner-6-issues
+  title: Limitations and known issues
+---
+
+You can use the IBM® Streams Runner for Apache Beam `FileStreamSample` sample application to learn how to use Bluemix object storage for file input and output.
+
+## Before you start
+
+Before you run the Apache Beam 2.0 `FileStreamSample` sample application, you must configure and run the following services on IBM Bluemix®:
+
+- Streaming Analytics. For more information, see [Creating a Streaming Analytics service on Bluemix](../beamrunner-2-install/#creating-a-streaming-analytics-service-on-bluemix).
+- Object Storage OpenStack Swift for Bluemix.
+   - Create the service if you don't already have one. For more information, see [Creating the Object Storage OpenStack Swift for Bluemix service](../beamrunner-5a-io/#creating-the-object-storage-openstack-swift-for-bluemix-service).
+   - Set up credentials for the service. **Remember**: Make sure the environment variables are configured. For more information, see [Set up credentials for the service](../beamrunner-5a-io/#setting-up-credentials-for-the-service).
+   - (Optional) Install the Swift CLI client. For more information, see [Configuring the CLI to use Swift and Cloud Foundry commands](https://console.bluemix.net/docs/services/ObjectStorage/os_configuring.html).
+
+**Important**: If you want to compile your application on Bluemix, you must unset the `STREAMS_INSTALL` variable before you submit the application to the Streaming Analytics service.
+
+## Running the sample application
+
+These instructions assume that you have already set up and run other samples on the Streaming Analytics service on Bluemix.
+
+1. Navigate to the `samples` directory in Streams Runner, and set up environment variables for the runner:
+
+    ```
+    cd <installdir>/samples
+    . bin/streams-runner-env.sh

```

Comment:
@pgerv12 is the `.` at the beginning of this line a typo?

Reply:
@mrshenli I don't believe so (it's a short annotation for `source`)